src/styles/styles.js

```diff
@@ -2510,6 +2510,40 @@ function getLoginPagePromoStyle() {
     return promos[_.random(0, 3)];
 }
 
+/**
+ * Generate the styles for the ReportActionItem wrapper view.
+ *
+ * @param {Boolean} [isHovered]
+ * @returns {Object}
+ */
+function getReportActionItemStyle(isHovered = false) {
```

Comment:
Why did we move these?

Reply:
I moved these because there wasn't a good reason to separate them and wanted to enforce the pattern of having these style helper methods imported like `import * as Styles from './styles/styles'` instead of having some of them be default exports.